src/libraries/System.Private.Runtime.InteropServices.JavaScript/src/ILLink/ILLink.Suppressions.wasm.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
This doesn't need to be addressed in this PR, but making this a `wasm.xml` file is a bit misleading since libraries outside of CoreLib don't build per architecture - but instead they build for OS:

[https://github.com/dotnet/runtime/blob/f4094bfaaa7e2fa3fd5b765949adb464a9bd4d73/src/libraries/System.Private.Runtime.InteropServices.JavaScript/src/System.Private.Runtime.InteropServices.JavaScript.csproj#L5](https://github.com/dotnet/runtime/blob/f4094bfaaa7e2fa3fd5b765949adb464a9bd4d73/src/libraries/System.Private.Runtime.InteropServices.JavaScript/src/System.Private.Runtime.InteropServices.JavaScript.csproj#L5)

Reply:
Thanks, will address now.